docs/integrations/destinations/local-json.md

```diff
@@ -44,5 +44,13 @@ The local mount is mounted by Docker onto `LOCAL_ROOT`. This means the `/local`
 * the local mount is using the `/tmp/airbyte_local` default
 * then all data will be written to `/tmp/airbyte_local/cars/models` directory.
 
-\(If Airbyte instance is running on the same computer that you are navigating with, you can open your browser to go to [file:///tmp/airbyte\_local](file:///tmp/airbyte_local) and look at the replicated data locally\)
+### Access Replicated Data Files
+
+* If Airbyte instance is running on the same computer that you are navigating with, you can open your browser to go to [file:///tmp/airbyte\_local](file:///tmp/airbyte_local) and look at the replicated data locally
+* If Airbyte instance is running on a remote server or for some reasons you can't use the first approach ([file:///tmp/airbyte\_local]) to access the replicated files, follow the following steps to access the replicated files:
+    * Access the scheduler container using `docker exec -it airbyte-scheduler bash`
+    * Make sure you are at the root folder of the container using `cd ..`
```

Comment:
```suggestion
```
using the next command you already go to the correct path, so you can remove this one